waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -161,20 +161,16 @@ genPublicDir :: AppSpec -> Generator [FileDraft]
 genPublicDir spec =
   return $
     extPublicFileDrafts
-      ++ ifUserDidntProvideFile genFaviconFd
       ++ ifUserDidntProvideFile genManifestFd
   where
     publicFiles = AS.externalPublicFiles spec
     extPublicFileDrafts = map C.mkPublicFileDraft publicFiles
-    genFaviconFd = C.mkTmplFd (C.asTmplFile [relfile|public/favicon.ico|])
     genManifestFd = C.mkTmplFdWithData tmplFile tmplData
```

Comment:
Old code, but this function's name is wrong - `gen` implies it's a generator monad but it's just a file draft.